db/model/generator.js

```diff
@@ -154,6 +155,45 @@ module.exports = function generator(seq, dataTypes) {
           override: true,
         });
       },
+
+      /**
+       * 1. validate the collectors field: if succeed, save the collectors in temo var for
+       *  attaching to generator. if fail, abort the POST operation
+       * 2. create the generator
+       * 3. add the saved collectors (if any)
+       */
+      createWithCollectors(requestBody, user) {
```

Comment:
looks like "user" is not used.

Reply:
good point. will remove in another commit